staging/src/k8s.io/apiserver/pkg/util/flag/flags.go

```diff
@@ -43,12 +43,16 @@ func WarnWordSepNormalizeFunc(f *pflag.FlagSet, name string) pflag.NormalizedNam
 	return pflag.NormalizedName(name)
 }
 
-// InitFlags normalizes, parses, then logs the command line flags
+// InitFlags normalizes and parses the command line flags
 func InitFlags() {
 	pflag.CommandLine.SetNormalizeFunc(WordSepNormalizeFunc)
 	pflag.CommandLine.AddGoFlagSet(goflag.CommandLine)
 	pflag.Parse()
-	pflag.VisitAll(func(flag *pflag.Flag) {
+}
+
+// PrintFlags logs the flags in the flagset
+func PrintFlags(flags *pflag.FlagSet) {
```

Comment:
I'd rather have this here and duplicated in k8s.io/kubernetes (near the verflag probably) than have everything depend on this from apiserver.

I'd make it V(1).

Reply:
thanks @deads2k will update the PR shortly